pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -404,6 +405,10 @@ func (c *Cluster) provisionPodTemplateSpec(osdProps osdProperties, restart v1.Re
 	if osdProps.pvc.ClaimName != "" {
 		// Create volume config for PVCs
 		volumes = append(volumes, getPVCOSDVolumes(&osdProps)...)
+		devVolume := v1.Volume{Name: "devices", VolumeSource: v1.VolumeSource{HostPath: &v1.HostPathVolumeSource{Path: "/dev"}}}
```

Comment:
Instead of adding these mounts here, could you just add a condition on line 397 above if we're configuring on a PVC?
`|| osdProps.pvc.ClaimName != ""`